caas/kubernetes/provider/bootstrap.go

```diff
@@ -24,8 +24,13 @@ import (
 	"github.com/juju/juju/mongo"
 )
 
-// JujuControllerStackName is the juju CAAS controller stack name.
-const JujuControllerStackName = "juju-controller"
+const (
+	// JujuControllerStackName is the juju CAAS controller stack name.
+	JujuControllerStackName = "juju-controller"
```

Comment:
We don't want the "juju-" prefix.
We have already removed it from operator and workload pods.

Reply:
changed `controller` now